### PR TITLE
Wip/troubleshooting

### PR DESCRIPTION
**Pull request by piknikki**
*What does this PR do?*
Fixing circle problems.

**Is this a feature or refactor?**
- Bug fixes with the size of the timer circle

**How should this be tested?**
- Add new activity and make sure changing frame size doesn't look horrible.

### CODE REVIEW

**Review by Cody-Durham — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h3>Select a category</h3>
                 Meditate
               </button>
             </div>
-            <p class='button-error hidden'><img class='warning' src='./assets/error-yellow.svg'>Please select a category</p>
+            <p class='button-error hidden'><img class='warning' src='./assets/error-yellow.svg' alt="error icon">Please select a category</p>
```
Comment (Cody-Durham) [inline index.html:37]: Great add for WAVE

File: index.html [hunk 2]
```diff
@@ -44,12 +44,12 @@ <h3>Select a category</h3>
               <div class='minutes'>
                 <label for='activity-minutes'>Minutes</label>
                 <input class='activity-minutes' id='activity-minutes' type='number' max='60'  min='0' placeholder='Enter minutes here' onkeydown="return false">
-                <p class='minutes-error hidden'><img class='warning' src='./assets/error-yellow.svg'>This field is required</p>
+                <p class='minutes-error hidden'><img class='warning' src='./assets/error-yellow.svg' alt="error icon">This field is required</p>
```
Comment (Cody-Durham) [inline index.html:47]: Great add for WAVE

File: index.html [hunk 3]
```diff
@@ -44,12 +44,12 @@ <h3>Select a category</h3>
...
               </div>
               <div class='seconds'>
                 <label for='activity-seconds'>Seconds</label>
                 <input class='activity-seconds' id='activity-seconds' type='number' max='59'  min='0' step='5' placeholder='Enter seconds here' onkeydown="return false">
-                <p class='seconds-error hidden'><img class='warning' src='./assets/error-yellow.svg'>This field is required</p>
+                <p class='seconds-error hidden'><img class='warning' src='./assets/error-yellow.svg' alt="error icon">This field is required</p>
```
Comment (Cody-Durham) [inline index.html:52]: Great add for WAVE

**Review by Cody-Durham — APPROVED** [review]

**Comment by Cody-Durham** [comment]
If we have time, we can try and troubleshoot a bit more on how to make the circle not get squished when adjusting the page size

File: index.html (hunk 1 above)
Comment (piknikki) [reply index.html:37]: 💥

**Pull request merged**